docs/orchestration/execution/storage_options.md

```diff
@@ -102,6 +102,20 @@ Additionally, in more recent releases of Core your flow will default to using a
 GCS Storage uses Google Cloud credentials the same way as the standard [google.cloud library](https://cloud.google.com/docs/authentication/production#auth-cloud-implicit-python) which means both upload (build) and download (local agent) times need to have the proper Google Application Credentials configuration.
 :::
 
+## GitHub
+
+[GitHub Storage](/api/latest/environments/storage.html#github) is a storage option that uploads flows to a GitHub repository as `.py` files. Flows stored using this option can be run by [local agents](/orchestration/agents/local.html) as long as the machine running the local agent is configured to pull from a git repo or by container-based agents using the method outlined [below](/orchestration/execution/storage_options.html#non-docker-storage-for-containerized-environments).
```

Comment:
There's no reason why these shouldn't work with other agents, right? Provided a `metadata.image` is set on the environment, any agent should be able to execute a flow stored in github storage?

Reply:
*Aside: this is one reason why I prefer line wraps for documentation - as written it's hard to comment on the exact line, since the single text-file line is a whole paragraph.*